### ISSUE TITLE
Improve Visibility of Active Threads

### ISSUE THREAD

**Issue by hitchhooker** [vector-im/element-meta#2417]
### Your use case

**Problem:**
In the current implementation of the desktop and web versions of Element, messages within threads do not appear in the main chat in chronological order. This makes it difficult to follow conversations that are part of threads as they are not integrated with the timeline of standalone messages.

**Suggested Enhancement:**
I propose an option to sort all messages, including those within threads, chronologically in the main chat view(option to bounce whole thread as latest). This would allow users to see messages from both threads and direct replies in the order they were posted, ensuring a seamless flow of conversation.

**Expected Benefit:**
Implementing this feature would enhance the user experience by providing a continuous and coherent view of all conversations. It would help users keep track of discussions more effectively, fostering better engagement and communication across the platform.

### Have you considered any alternatives?

i tried to search such option in the logs or prefer checking mobile app element x where this is non issue.

**Comment by dbkr** [comment]
I'm attempting to triage this but I'm afraid I don't understand what's being proposed. Why would a message in a thread appear in the main chat at all? If they're out of order, wouldn't this be a bug? What does the title mean - is the 'feat' suggestion it's a feature? We don't use this naming convention, we use the labels.

**Comment by hitchho** [comment]
hey @dbkr,

so issue that we are facing using element for collaboration that when someone replies to lets say active thread from a week ago there is barely no way from UI to notice that someone has responded to the thread. 
I am asking for a flag/feature to be able to have threads bounce latest in message list when they get responses instead of staying in the location based on first message of the thread.

**Comment by thomasfrobieter** [comment]
I agree, we have the same problem and thats why we barely use this feature. The only indicator for new messages is the tiny badge on the topics icon.

This here is related: [#2510](https://github.com/element-hq/element-meta/issues/2510). IMO it is not necessary to move the whole thread, but all people (which are involved in the thread) should see some kind of 'log entry' in the main chat "New message in thread x", which opens the thread on click.
